lib/Traits/TraitsClasses/CatalogMemberTraits.ts

```diff
@@ -107,15 +107,19 @@ class CatalogMemberTraits extends ModelTraits {
     name: "Name",
     description: "The name of the catalog item."
   })
-  name?: string;
+  get name(): string | undefined {
+    return;
```

Comment:
If I read "return" in a getter, I expect some value to be returned, so when skimming code it would be quite easy to miss that this returns undefined despite being "implemented".

Would `get name(): string | undefined {}` give the same effect? That would make these "bogus-getters" more visually similar to the `name?: string` they replace, and different from a "real" getter.

Reply:
In typescript, getters have to return a value, if not TS2378 will be thrown
```
A 'get' accessor must return a value. ts(2378)
```
Maybe it can be made clearerr that the default value is undefined by using `return undefined`.

Reply:
Yes, @zoran995 is right. A return statement for getter is enforced by TS.